src/test/java/uk/gov/hmcts/reform/sendletter/model/WithServiceNameTest.java

```diff
@@ -30,8 +30,8 @@ public void should_serialize_as_expected() throws Exception {
 
         // then
         assertThat(json)
-            .hasJsonPathValue("@.template")
-            .hasJsonPathMapValue("@.values")
+            .hasJsonPathArrayValue("@.documents")
+            .hasJsonPathValue("@.type")
```

Comment:
may be worth to also checking properties of the nested objects

Reply:
I think we are validating if the json unwrapped is being applied correctly.I am not sure what value we would get if check for values